pretext/pretext.cfg

```diff
@@ -58,6 +58,7 @@ pdflatex = pdflatex
 xelatex = xelatex
 pdfsvg = pdf2svg
 asy = asy
+mermaid = mmdc
```

Comment:
CLI will need updates at e.g. https://github.com/PreTeXtBook/pretext-cli/blob/0fbcef135b56e92c9f6e89d75175848973fa6139/pretext/project/xml.py#L11

Reply:
CLI PR is https://github.com/PreTeXtBook/pretext-cli/pull/750

Reply:
Somehow I missed that part of the PR, great!